pkg/antctl/transform/appliedtogroup/transform.go

```diff
@@ -55,3 +58,38 @@ func Transform(reader io.Reader, single bool) (interface{}, error) {
 		listTransform,
 	)(reader, single)
 }
+
+var _ common.TableOutput = new(Response)
+
+func (r Response) GetTableHeader() ([]string, error) {
+	return common.GetTableHeader(r)
+}
+
+func (r Response) GetTableRow(args []string, maxColumnLength int) (table.Row, error) {
+	val := reflect.ValueOf(r)
+	var row table.Row
+	for _, arg := range args {
+		ele := val.FieldByName(arg).Interface()
+		switch arg {
+		case "Pods":
```

Comment:
Ditto
With the header content above, you can probably just construct several string here based on its content, without having to involve reflect and many type conversion.
```
func (r Response) GetTableRow(maxColumnLength int) []string {
        pods := ...
        return []string{r.Name, pods}
}
```